ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/onboarding/view/OnboardingView.kt

```diff
@@ -66,12 +67,16 @@ fun ComposablePreview() {
 }
 
 /**
+ * @param isFullOnboardingEnabled Feature toggle to control whether the full onboarding flow is enabled.  If disabled, then an abbreviated flow is shown
+ * and the onboarding state is treated effectively as if it is [OnboardingStage.Wallet].
  * @param onImportWallet Callback when the user decides to import an existing wallet.
  * @param onCreateWallet Callback when the user decides to create a new wallet.
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
+@OptIn(ExperimentalMaterial3Api::class)
+@Suppress("LongParameterList")
 fun Onboarding(
+    isFullOnboardingEnabled: Boolean,
```

Comment:
We have two parameters being passed in that can be somewhat orthogonal: the flag for whether the full onboarding experience is enabled, and the stage of onboarding the user should be shown.  I considered creating an entirely different set of composables to avoid this, but that would introduce hundreds more lines of code between the new composables and new tests.  This boolean seemed like the simplest solution for now until we see how this evolves over time.